packages/lab/src/stepper-input/StepperInput.tsx

```diff
@@ -59,13 +59,14 @@ export interface StepperInputProps
   /**
    * The value of the stepper input. The component will be controlled if this prop is provided.
    */
-  value?: number | string;
+  value?: number;
 }
 
 export const StepperInput = forwardRef<HTMLDivElement, StepperInputProps>(
   function StepperInput(props, ref) {
     const {
       className,
+      decimalPlaces,
```

Comment:
destruct but not other change? why?

Reply:
there was an error in console when it was passed to the input

Reply:
Is this prop expect to work with the component? 
Currently this is swallowed and take no effect?

If this meant to work, some of the props may need to go to `inputProps` rather than `Input` top level div.

Reply:
it does work as expected: https://9b4b76d8.saltdesignsystem-storybook.pages.dev/?path=/story/lab-stepper-input--decimal-places&args=decimalPlaces:4

Reply:
it's still used in usestepperinput hook

Reply:
This should trigger unused variable warning? Need to refactor how props are passed around. `decimalPlaces` warning likely means other props will have similar issue.

Reply:
I've updated this to destruct more props so only input props get passed on input